packages/dynamite/dynamite/example/lib/petstore.openapi.dart

```diff
@@ -1,10 +1,25 @@
+// Use of this source code is governed by a Apache 2.0 license. It can be obtained at https://www.apache.org/licenses/LICENSE-2.0.html.
+
 // OpenAPI client generated by Dynamite. Do not manually edit this file.
 
 // ignore_for_file: camel_case_types, discarded_futures
 // ignore_for_file: no_leading_underscores_for_local_identifiers
 // ignore_for_file: public_member_api_docs, unreachable_switch_case
 
-// ignore_for_file: no_leading_underscores_for_library_prefixes
+/// Swagger Petstore Version: 1.0.0.
+///
+/// A sample API that uses a petstore as an example to demonstrate features in the OpenAPI 3.0 specification.
+///
+/// You can contact the Swagger API Team team under:
+///   Email: `apiteam@swagger.io`.
+///   Website: `http://swagger.io`.
+///
+/// Use of this source code is governed by a Apache 2.0 license.
+/// It can be obtained at https://www.apache.org/licenses/LICENSE-2.0.html.
+///
+/// Usage of these apis must adhere to the terms of service: `http://swagger.io/terms/`.
+library; // ignore_for_file: no_leading_underscores_for_library_prefixes
```

Comment:
This is ugly. I'll open an issue (maybe PR) upstream.
But not a blocker for this PR.

Reply:
Reported upstream https://github.com/dart-lang/code_builder/pull/448

Reply:
Shall we wait for it?